typescript/infra/config/warp.ts

```diff
@@ -151,8 +153,9 @@ export const warpConfigGetterMap: Record<string, WarpConfigGetter> = {
   [WarpRouteIds.EthereumLineaTURTLE]: getEthereumLineaTurtleWarpConfig,
   [WarpRouteIds.ArbitrumAvalancheBaseBscEthereumLumiaprismOptimismPolygonLUMIA]:
     getArbitrumAvalancheBaseBscEthereumLumiaprismOptimismPolygonLUMIAWarpConfig,
-  // Not present in the registry
-  // [WarpRouteIds.TestnetCCTP]: getCCTPWarpConfig,
+  [WarpRouteIds.TestnetCCTP]: getTestnetCCTPWarpConfig,
```

Comment:
is this route in the registry? we commented it out because we ran into issues before with it being defined here but not in reg